lib/plugin.js

```diff
@@ -24,6 +24,11 @@ ManifestPlugin.prototype.getFileType = function(str) {
   return ext;
 };
 
+ManifestPlugin.prototype.existNameInManifest = function(name, manifest) {
```

Comment:
To be honest, this is the point in which my knowledge of webpack start shortening. I am not sure if it makes sense neither.

Something i did notice when debugging was that files added by copy plugin were never coming in `compilation.chunks`, but in `compilation.stats.assets`. Does this answer your question?

Unfortunately I could not extract much knowledge from issue #55 too many use cases i don't know at all. Which part do you think will conflict?